README.md

```diff
@@ -90,6 +90,7 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 - Wileyfox Swift - crackling
 - Xiaomi Mi 4i - ferrari
 - Xiaomi Redmi 2 - wt86047, wt88047
+- Yamada EveryPad III
```

Comment:
Can you explain this in the commit message, if this was an intentional change?

Reply:
This is the exact same device.